src/v2/index.js

```diff
@@ -88,11 +88,20 @@ const requestLogger = isProd
   })
   : morgan('dev');
 
-graphQLServer.use('*', helmet({
-  frameguard: false,
-  noSniff: false,
-  xssFilter: false,
-}), noCache(), requestLogger, cookieParser());
+if (process.env.NODE_ENV === 'development') {
+  graphQLServer.use('*', helmet({
+    frameguard: false,
+    noSniff: false,
+    xssFilter: false,
+    contentSecurityPolicy: false, // adding this for graphQL api
+  }), noCache(), requestLogger, cookieParser());
+} else {
+  graphQLServer.use('*', helmet({
+    frameguard: false,
+    noSniff: false,
+    xssFilter: false,
+  }), noCache(), requestLogger, cookieParser());
+}
```

Comment:
This should do the same with simpler code.
```suggestion
  graphQLServer.use('*', helmet({
    frameguard: false,
    noSniff: false,
    xssFilter: false,
    contentSecurityPolicy: process.env.NODE_ENV === 'production' // False for development
  }), noCache(), requestLogger, cookieParser());
```